netdata.spec.in

```diff
@@ -691,7 +691,7 @@ fi
 %{_libexecdir}/%{name}/plugins.d/charts.d.plugin
 %{_libexecdir}/%{name}/plugins.d/charts.d.dryrun-helper.sh
 %{_libexecdir}/%{name}/charts.d/
-%defattr(0644,root,netdata,0644)
+%defattr(0644,root,netdata,0650)
```

Comment:
Pretty sure the directory permissions should actually be `0755` here and not `0650`. Same for the other ones as well.

Reply:
Done